cluster/addons/fluentd-elasticsearch/fluentd-es-image/Gemfile

```diff
@@ -1,11 +1,11 @@
 source 'https://rubygems.org'
 
-gem 'fluentd', '~>0.12.32'
-gem 'activesupport', '~>4.2.6'
-gem 'fluent-plugin-kubernetes_metadata_filter', '~>0.27.0'
-gem 'fluent-plugin-elasticsearch', '~>1.9.5'
-gem 'fluent-plugin-systemd', '~>0.0.8'
-gem 'fluent-plugin-detect-exceptions', '~>0.0.8'
+gem 'fluentd', '<=1.1.0'
+gem 'activesupport', '~>5.1.4'
+gem 'fluent-plugin-kubernetes_metadata_filter', '~>1.0.0'
+gem 'fluent-plugin-elasticsearch', '~>2.4.1'
+gem 'fluent-plugin-systemd', '~>0.3.1'
+gem 'fluent-plugin-detect-exceptions', git: 'https://github.com/monotek/fluent-plugin-detect-exceptions'
```

Comment:
Can you change this back to `'~>0.0.9'` now that [it is released](https://github.com/GoogleCloudPlatform/fluent-plugin-detect-exceptions/releases/tag/v0.0.9)